src/Metadata/AbstractContainer.php

```diff
@@ -18,11 +18,11 @@
 abstract class AbstractContainer
 {
     /**
-     * @var MetadataParser
+     * @var MetadataReaderWriter
      */
     protected $parser;
```

Comment:
Shall we rename the variable as well to `$metadataReaderWriter`?

Reply:
Done 0f697407125b0c5218cfbb80cd2b132dc3ace75e